docs/source/conf-datasources.rst

```diff
@@ -25,39 +25,28 @@ Example:
   {
     "data_sources": {
       "kinesis": {
-        "abc_corporate_stream_alert_kinesis": {
-          "logs": [
-            "box",
-            "pan"
-          ]
-        },
-        "abc_production_stream_stream_alert_kinesis": {
-          "logs": [
-            "inspec",
-            "osquery"
-          ]
-        }
-      },
+        "abc_corporate_stream_alert_kinesis": [
```

Comment:
please update these to remove the `_kinesis` suffix. this format is no longer the naming convention